Scripts/RemediationScripts/Remediate-SetServiceBusMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,806 @@
+        Write-Host $([Constants]::SingleDashLine)
+
+        # Loop through the list of Azure Service Bus Namespace(s) which needs to be remediated.
+        $actionableServiceBusNamespacesDetails  | ForEach-Object {
+            try {
+                $updatedTLSVersion = (Set-AzServiceBusNamespace -ResourceGroupName $_.ResourceGroupName -NamespaceName $_.ResourceName -MinimumTlsVersion $([Constants]::MinRequiredTLSVersionValue)).MinimumTlsVersion 
+
+                if (CheckIfCurrentTLSVersionIsSecure($updatedTLSVersion)) {
+                    $_.MinimumTLSVersion = $updatedTLSVersion
+                    $serviceBusNamespacesRemediated += $_
+                    $logResource = @{}	
+                    $logResource.Add("ResourceGroupName", ($_.ResourceGroupName))	
+                    $logResource.Add("ResourceName", ($_.ResourceName))	
+                    $logRemediatedResources += $logResource	
+                }
+                else {
+                    $serviceBusNamespacesSkipped += $_
+                    $logResource = @{}	
+                    $logResource.Add("ResourceGroupName", ($_.ResourceGroupName))	
+                    $logResource.Add("ResourceName", ($_.ResourceName))
+                    $logResource.Add("Reason", "Error while setting MinimumTLSVersion: [$($_)]")            
+                    $logSkippedResources += $logResource	
+
+                }                
+            }
+            catch {
+                $serviceBusNamespacesSkipped += $_
+                $logResource = @{}	
+                $logResource.Add("ResourceGroupName", ($_.ResourceGroupName))	
+                $logResource.Add("ResourceName", ($_.ResourceName))	
+                $logResource.Add("Reason", "Encountered error while setting MinimumTLSVersion")    	
+                $logSkippedResources += $logResource
+            }
+        }
+
+        Write-Host $([Constants]::DoubleDashLine)
+
+
+        if ($AutoRemediation) {
+            if ($($serviceBusNamespacesRemediated | Measure-Object).Count -gt 0) {
+
+                # Write this to a file.
+                $serviceBusNamespacesRemediatedFile = "$($backupFolderPath)\RemediatedServiceBusNamespaces.csv"
+                $serviceBusNamespacesRemediated | Export-CSV -Path $serviceBusNamespacesRemediatedFile -NoTypeInformation
+
+                Write-Host "The information related to Azure Service Bus Namespace(s) where Minimum TLS version was set, has been saved to [$($serviceBusNamespacesRemediatedFile)]. Use this file for any roll back that may be required." -ForegroundColor $([Constants]::MessageType.Update)
+                Write-Host $([Constants]::SingleDashLine)
+            }
+
+            if ($($serviceBusNamespacesSkipped | Measure-Object).Count -gt 0) {
+                $serviceBusNamespacesSkipped | Format-Table -Property $colsProperty -Wrap            
```

Comment:
there is no need to print table in AutoRemediation mode

Reply:
Addressed.